### PR TITLE
Resolve MUWM-3942 - handle eval not exist case.

### PR DESCRIPTION
**Pull request by fanglinfang**
https://jira.cac.washington.edu/browse/MUWM-3942

### CODE REVIEW

**Comment by klpittman** [comment]
myuw__teaching

This is what I'm seeing.

**Comment by fanglinfang** [comment]
CM 421 A and ACCTG 508 A are the one with the mock data for this case.
We display the eval not set up message.

File: myuw/views/api/instructor_schedule.py [hunk 1]
```diff
@@ -110,15 +110,18 @@ def set_section_evaluation(section, person):
...
         evaluations = get_evaluation_by_section_and_instructor(
             section, person.employee_id)
-        for eval in evaluations:
-            if eval.section_sln == section.sln:
-                return eval.json_data()
-    except DataFailureException as ex:
-        # eval search never returns 404
+        if evaluations is not None:
+            for eval in evaluations:
+                print eval.json_data()
```
Comment (mikeseibel) [inline myuw/views/api/instructor_schedule.py]: maybe some stray debugging?

Comment (fanglinfang) [inline myuw/views/api/instructor_schedule.py]: oops. missed that one.

**Pull request merged**